pages/api/auth/[...nextauth].ts

```diff
@@ -1,11 +1,36 @@
 import NextAuth from "next-auth/next";
 import GoogleProvider from "next-auth/providers/google"
 
-export default NextAuth({
+// export default NextAuth({        //set up NextAuth to handle authentication, after google login redirects to this API route
```

Comment:
I'm not sure you need this massive comment?

Reply:
It was just the code I had previously before I changed it, in case the change didn't work out